### DISCUSSION TITLE
[@types/pouchdb-mapreduce] Lacks proper type for `PouchDB.Query.Response` `.rows[*].value`

### DISCUSSION THREAD

**Discussion by matthew-holder-revvity** [DefinitelyTyped/DefinitelyTyped#71264] [Issues with a @types package]
The type for `PouchDB.Query.Response` `.rows[*].value` should be `Content | null` and not `any`, since the `value: Content` from `emit` is what is assigned.

Also, the second argument to `emit` is optional, resulting in `null` for a row's value.

[a comment by an automated account is not shown]

**Comment by geppy** [comment]
@matthew-holder-revvity Did you want to open a PR for this?

**Reply by matthew-holder-revvity** [reply]
I might be able to after the holidays.

**Comment by matthew-holder-revvity** [comment]
Sorry forgot about it after the holidays.
